src/FieldElementSubsystems.Test/Point/PointTest.cs

```diff
@@ -24,7 +25,7 @@ public void PointShouldParseConfiguration()
             .AddInMemoryCollection(testSettings)
             .Build();
 
-        var point = new EulynxLive.Point.Point(Mock.Of<ILogger<EulynxLive.Point.Point>>(), configuration);
+        var point = new EulynxLive.Point.Point(Mock.Of<ILogger<EulynxLive.Point.Point>>(), configuration, Mock.Of<IPointToInterlockingConnection>());

```

Comment:
we should consider adding a few more tests with every PR as we go along.

Reply:
#16